### PR TITLE
chore(changelog): update changelog for v0.11.0 release

### PR DESCRIPTION
**Pull request by Pouyanpi**
## Description

v0.11.0 changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,41 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 NOTE:
 The changes related to the Colang language and runtime have moved to [CHANGELOG-Colang](./CHANGELOG-Colang.md) file.
 
+## [0.11.0] - 2024-11-15
+
+### Added
+
+- **Observability**: Add observability support with support for different backends ([#844](https://github.com/NVIDIA/NeMo-Guardrails/pull/844)) by @Pouyanpi
+- **Passthrough Support**: Add passthrough support in Colang 2
```
Comment (schuellc-nvidia) [inline CHANGELOG.md]: Should we move that to Colang 2 changelog?

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,41 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+- **Langchain 3 support**: Upgrade LangChain to Version 0.3 ([#784](https://github.com/NVIDIA/NeMo-Guardrails/pull/784)) by @Pouyanpi
+- **Sandboxed Environment in Jinja2**: Add sandboxed environment in Jinja2 ([#799](https://github.com/NVIDIA/NeMo-Guardrails/pull/799)) by @Pouyanpi
+- **vllm**: Bump vllm from 0.2.7 to 0.5.5 for llama_guard and patronusai([#836](https://github.com/NVIDIA/NeMo-Guardrails/pull/836)) by @app/dependabot
+- **Python 3.8**: Drop support for Python 3.8 ([#803](https://github.com/NVIDIA/NeMo-Guardrails/pull/803)) by @Pouyanpi
+
+### Fixed
+
+- **Streaming*: Handle multiple output parsers in generation ([#854](https://github.com/NVIDIA/NeMo-Guardrails/pull/854)) by @Pouyanpi
+- **retrieve chunk action**: Fix presence of new line in retrieve chunk action ([#809](https://github.com/NVIDIA/NeMo-Guardrails/pull/809)) by @Pouyanpi
+- **Standard Library import**: Fix guardrails standard library import path in Colang 2.0 ([#835](https://github.com/NVIDIA/NeMo-Guardrails/pull/835)) by @Pouyanpi
```
Comment (schuellc-nvidia) [inline CHANGELOG.md:32]: Move to Colang 2 changelog

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,41 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+- **tests**: Mock PromptSession to prevent console error ([#851](https://github.com/NVIDIA/NeMo-Guardrails/pull/851)) by @Pouyanpi
+
+### Documentation
+
+- **Documentation Restructuring**: Restructure the docs and several style enhancements ([#855](https://github.com/NVIDIA/NeMo-Guardrails/pull/855)) by @Pouyanpi
+- **installation**: Update optional dependencies install ([#853](https://github.com/NVIDIA/NeMo-Guardrails/pull/853)) by @Pouyanpi
+- **role**: Update role from bot to assistant ([#852](https://github.com/NVIDIA/NeMo-Guardrails/pull/852)) by @Pouyanpi
+- **Got It AI deprecation**: Add deprecation notice for Got It AI integration ([#857](https://github.com/NVIDIA/NeMo-Guardrails/pull/857)) by @mlmonk
+
 ## [0.10.0] - 2024-09-23
```
Comment (schuellc-nvidia) [inline CHANGELOG.md]: I changed that to 2024-09-27 in the Colang 2 changelog PR #856

File: CHANGELOG.md (hunk 1 above)
Comment (Pouyanpi) [reply CHANGELOG.md]: Sure, let's move it.

File: CHANGELOG.md (hunk 2 above)
Comment (Pouyanpi) [reply CHANGELOG.md:32]: this concerns guardrails, it is the guardrails standard library import, should we move it?

**Comment by Pouyanpi** [comment]
- [ ] railsignore to changelog and release notes

**Pull request merged**